src/components/mobile/mobileAssetView.tsx

```diff
@@ -0,0 +1,302 @@
+import { Callout, Classes } from "@blueprintjs/core";
```

Comment:
There might be some CP/decommission/etc. code in here that is reused from AssetView, but that's mostly because i don't fully understand the rendering logic involving CP/decommission/any other new features we've implemented and didn't want to break anything. It seems to work fine though.

Reply:
the user will have no way to enter a change plan from mobile so you should be safe to remove all that logic